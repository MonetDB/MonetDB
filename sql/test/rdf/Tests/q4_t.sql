--Query4:
SELECT B.prop, B.obj, count(*)
	FROM triples_pso AS A,
	triples_pso AS B,
	triples_pso AS C
	WHERE A.subj = B.subj
	AND A.prop = 17582582
	AND A.obj = 14660332
--       AND A.prop = '<type>'
--       AND A.obj = '<Text>'
	AND C.subj = B.subj
	AND C.prop = 14659603
	AND C.obj = 17106461
--       AND C.prop = '<language>'
--       AND C.obj =  '<language/iso639-2b/fre>'
	GROUP BY B.prop, B.obj
	HAVING count(*) > 1;

