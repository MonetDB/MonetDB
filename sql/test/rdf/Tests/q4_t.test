query III rowsort
SELECT B.prop, B.obj, count(*)
	FROM triples_pso AS A,
	triples_pso AS B,
	triples_pso AS C
	WHERE A.subj = B.subj
	AND A.prop = 17582582
	AND A.obj = 14660332
	AND C.subj = B.subj
	AND C.prop = 14659603
	AND C.obj = 17106461
	GROUP BY B.prop, B.obj
	HAVING count(*) > 1
----

