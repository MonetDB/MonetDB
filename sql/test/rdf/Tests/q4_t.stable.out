stdout of test 'q4_t` in directory 'sql/test/rdf` itself:


# 10:46:22 >  
# 10:46:22 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36263" "--set" "mapi_usock=/var/tmp/mtest-3472/.s.monetdb.36263" "--set" "monet_prompt=" "--forcemito" "--dbpath=/ufs/sjoerd/@Monet-stable/var/MonetDB/mTests_sql_test_rdf"
# 10:46:22 >  

# MonetDB 5 server v11.27.12 (hg id: 9a05add39d7b)
# This is an unreleased version
# Serving database 'mTests_sql_test_rdf', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 62.695 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2018 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://methuselah.da.cwi.nl:36263/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-3472/.s.monetdb.36263
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded


# 10:46:22 >  
# 10:46:22 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-3472" "--port=36263"
# 10:46:22 >  

#SELECT B.prop, B.obj, count(*)
#	FROM triples_pso AS A,
#	triples_pso AS B,
#	triples_pso AS C
#	WHERE A.subj = B.subj
#	AND A.prop = 17582582
#	AND A.obj = 14660332
#--       AND A.prop = '<type>'
#--       AND A.obj = '<Text>'
#	AND C.subj = B.subj
#	AND C.prop = 14659603
#	AND C.obj = 17106461
#--       AND C.prop = '<language>'
#--       AND C.obj =  '<language/iso639-2b/fre>'
#	GROUP BY B.prop, B.obj
#	HAVING count(*) > 1;
% sys.b,	sys.b,	sys.%1 # table_name
% prop,	obj,	%1 # name
% int,	int,	bigint # type
% 1,	1,	1 # length

# 10:46:22 >  
# 10:46:22 >  "Done."
# 10:46:22 >  

